src/features/click-to-load.js

```diff
@@ -473,23 +482,53 @@ class DuckWidget {
                     if (onError) {
                         fbElement.addEventListener('error', onError, { once: true })
                     }
+                // eslint-disable-next-line promise/prefer-await-to-then
+                }).catch((err) => {
+                    if (err === 'ddg-ctp-user-cancel') {
```

Comment:
I wonder if we should re-throw the exception if it's not the user cancelling one?

Also, where is this being thrown? I can't find `ddg-ctp-user-cancel` elsewhere. I was looking for it because I wondered if the exception is _really_ a string, I figured it might be something like `new Error("ddg-ctp-user-cancel")` instead.